src/main/java/com/box/sdk/BoxTransactionalAPIConnection.java

```diff
@@ -0,0 +1,161 @@
+     * @param  clientSecret the client secret to use when refreshing the access token.
+     * @param  accessToken  an initial access token to use for authenticating with the API.
+     * @param  refreshToken an initial refresh token to use when refreshing the access token.
+     * @throws BoxAPIException Box Transactional API does not support creating an connection with these params
+     */
+    public BoxTransactionalAPIConnection(String clientID, String clientSecret, String accessToken,
+                                            String refreshToken) {
+        super(null);
+        throw new BoxAPIException("This constructor is not available for BoxTransactionalAPIConnection.");
+    }
+
+    /**
+     * Disabling an invalid constructor for Box Developer Edition.
+     * @param  clientID     the client ID to use when exchanging the auth code for an access token.
+     * @param  clientSecret the client secret to use when exchanging the auth code for an access token.
+     * @param  authCode     an auth code obtained from the first half of the OAuth process.
+     * @throws BoxAPIException Box Transactional API does not support creating an connection with these params
+     */
+    public BoxTransactionalAPIConnection(String clientID, String clientSecret, String authCode) {
+        super(null);
+        throw new BoxAPIException("This constructor is not available for BoxTransactionalAPIConnection.");
+    }
+
+    /**
+     * Disabling an invalid constructor for Box Developer Edition.
+     * @param  clientID     the client ID to use when requesting an access token.
+     * @param  clientSecret the client secret to use when requesting an access token.
+     * @throws BoxAPIException Box Transactional API does not support creating an connection with these params
+     */
+    public BoxTransactionalAPIConnection(String clientID, String clientSecret) {
+        super(null);
+        throw new BoxAPIException("This constructor is not available for BoxTransactionalAPIConnection.");
+    }
+
+    /**
+     * Request a scoped transactional token.
+     * @param accessToken application access token.
+     * @param scope scope of transactional token.
+     * @return a BoxAPIConnection which can be used to perform transactional requests.
+     */
+    public static BoxAPIConnection getTransactionConnection(String accessToken, String scope) {
+        return BoxTransactionalAPIConnection.getTransactionConnection(accessToken, scope, null);
+    }
+
+    /**
+     * Request a scoped transactional token for a particular resource.
+     * @param accessToken application access token.
+     * @param scope scope of transactional token.
+     * @param resource resource transactional token has access to.
+     * @return a BoxAPIConnection which can be used to perform transactional requests.
+     */
+    public static BoxAPIConnection getTransactionConnection(String accessToken, String scope, String resource) {
```

Comment:
@gcurtis I like that too, although I'm somewhat hesitant to litter the transactional auth code throughout the SDK, especially since its primary use case is api calls from the browser. Imo, there isn't a whole lot of utility for scoped tokens on the server side, you may as well just use the fully privileged token for the call, and avoid making the downscope api call